core/models.py

```diff
@@ -42,3 +42,15 @@ class GlobalMetrics(models.Model):
     class Meta:
         verbose_name = "Global Metric"
         verbose_name_plural = "Global Metrics"
+
+class Suggestions(models.Model):
+    first_name = models.CharField(max_length=50)
+    last_name = models.CharField(max_length=50)
+    suggestion = models.TextField(max_length=500)
+
```

Comment:
looks good, lets also capture their email incase we want to start a correspondence